package-res/resources/web/pentaho/type/property.js

```diff
@@ -107,14 +107,14 @@ define([
         /**
          * Setting name and label first allows describing the property in subsequent error messages.
          *
-         * Setting label after name ensures that label defaults that are derived from name work the first time.
+         * Setting label after name ensures that label defaults that are derived from name will work the first time.
          *
          * Setting type before value (not included here, so it is processed after type) avoids:
          * 1. checking the new value against the old type (and then again, against the new type)
          * 2. ensures error messages are given in a predicatable order,
          *    independently of the order of properties in an instSpec:
-         *   1. is new type a subtype of old type?
-         *   2. is new value an instance of the new type?
+         *   a. is new type a subtype of old type?
+         *   b. is new value an instance of the new type?
```

Comment:
No, we cannot replace 1. -> a. in nested lists, because this is markdown notation. The doc build will most likely transform the numbering in the sub-list to something that makes sense